src/Template/Element/action-header.ctp

```diff
@@ -8,4 +8,5 @@ if (!$this->exists('actions')) {
     $this->end();
 }
 ?>
-<h2><?= $this->get('title'); ?><span class="actions"><?= $this->fetch('actions'); ?></span></h2>
```

Comment:
What if a record has a long ass title and one is checking on a smaller screen resolution?